simulation_parameters/SimulationParameters.cs

```diff
@@ -332,6 +336,21 @@ public IEnumerable<DifficultyPreset> GetAllDifficultyPresets()
         return difficultyPresets.Values;
     }
 
+    public ScreenEffect GetScreenEffect(string name)
+    {
+        return screenEffects[name];
+    }
+
+    public ScreenEffect GetScreenEffectByIndex(int index)
+    {
+        return screenEffects.Values.First(p => p.Index == index);
```

Comment:
I'm sure there's a better way to do this than abuse LINQ like this...

Reply:
In this code's defense, the same exact approach is used in other places e.g. the difficulty presets.

Reply:
Also, I may or may not have stolen that code from the difficulty presets.

Reply:
I see now, "index" isn't literally the index here but just a random value that could be whatever so this is actually kind of needed to guard against that. I'd prefer using programming terms with their usual meaning. That would require a bit of a refactor here, though (or renaming Index to Order).